projects/lights_control/inc/lights_events.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+typedef enum {
+  // front board events
+  LIGHTS_EVENT_HORN = 0,
+  LIGHTS_EVENT_HEADLIGHTS,
+  // rear board events
+  LIGHTS_EVENT_BRAKES,
+  LIGHTS_EVENT_STROBE,
+  // both boards
+  LIGHTS_EVENT_SIGNAL_LEFT,
+  LIGHTS_EVENT_SIGNAL_RIGHT,
+  LIGHTS_EVENT_SIGNAL_HAZARD,
+  LIGHTS_EVENT_SYNC,
+  // can internal events
+  LIGHTS_EVENT_CAN_RX,
```

Comment:
Note that the `event_queue` is a priority queue so you may want to make CAN related IDs higher priority.